### ISSUE TITLE
Use labels for partitions

### ISSUE THREAD

**Issue by ekultails** [ekultails/mac-linux-gaming-stick#71]
This makes it more reliable than UUIDs since we reset the machine-id. Currently, some machines fail to boot with the latest builds because the partition UUIDs are not found.

Error:

```
ERROR: device '635c62d0-a875-4996-97af-83c12349cfe8' not found. Skipping fsck.
mount: /new_root: can't find UUID=635c62d0-a875-4996-97af-83c12349cfe8.
```

Fix:

Set labels for all of the partitions. Then generate the fstab entries using the `-L` argument:

`$ genfstab -L -p /mnt > /mnt/etc/fstab`

**Comment by ekultails** [comment]
GRUB 2 is the real issue here and it does not appear to natively support labels. However, there is a workaround: https://askbot.fedoraproject.org/en/question/29209/how-do-i-set-grub2-to-use-labels-versus-uuid-in-search/

**Comment by ekultails** [comment]
There is now a new error:

```
ERROR: device '/dev/vda3' not found. Skipping fsck.
mount: /new_root: special device /dev/vda3 does not exist.
```

**Comment by ekultails** [comment]
The problem was found to actually be that a MacBook Pro 16" 2019 does not recognize the drive after the initial boot into the GRUB menu. The label workaround may actually not be required. Booting does not work with Linux 5.4 or even 5.10.

**Comment by ekultails** [comment]
The Mac booting problems may be resolved by: 04f859e09f53a62dd604f6125497a225af86ec36
